project/src/main/java/com/google/sps/data/Popular.java

```diff
@@ -26,41 +26,43 @@
 
 /** Class containing restaurants' popularity scores (# of times 'favorited'). */
 public final class Popular {
-  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  Query query = new Query("Popular");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Popular");
```

Comment:
Instead of doing this, why don't we use Query.Filter so we don't have to process it on the result?

https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/Query.html?hl=en#setfilter